plugin/pkg/admission/persistentvolume/label/admission.go

```diff
@@ -75,6 +77,13 @@ func (l *persistentVolumeLabel) Admit(a admission.Attributes) (err error) {
 		}
 		volumeLabels = labels
 	}
+	if volume.Spec.GCEPersistentDisk != nil {
+		labels, err := l.findGCEPDLabels(volume)
+		if err != nil {
+			return admission.NewForbidden(a, fmt.Errorf("error querying GCE PD volume %s: %v", volume.Spec.GCEPersistentDisk.PDName, err))
```

Comment:
I opened #20564 to track the lack of e2e coverage and #20955 to track the AWS problem, but I didn't realize this had broken it on GCE also.  Sorry!

For AWS, I was thinking of creating a volume with a well-known dummy name, which would then be ignored.  But that is pretty evil...